website/content/commands/acl/role/update.mdx

```diff
@@ -7,12 +7,22 @@ page_title: 'Commands: ACL Role Update'
 
 Command: `consul acl role update`
 
+Corresponding HTTP API Endpoint: [\[PUT\] /v1/acl/role/:id](https://www.consul.io/api-docs/acl/roles#update-a-role)
```

Comment:
This is another one that has two corresponding API endpoints:
- https://www.consul.io/api-docs/acl/roles#read-a-role
- https://www.consul.io/api-docs/acl/roles#read-a-role-by-name

We should resolve the same way as the above comment (if any changes are made).